src/bin/wast.rs

```diff
@@ -168,7 +168,7 @@ fn main() {
         wast_context
             .run_file(Path::new(&filename))
             .unwrap_or_else(|e| {
-                eprintln!("{}", e);
+                eprintln!("{:?}", e);
```

Comment:
Mind prefixing this with `error: ` as well to clearly indicate that it's an error?

Reply:
Sorry, glanced at this but promptly forgot.